pkgs/version/version.go

```diff
@@ -20,8 +20,7 @@ func CheckRc(s string) bool {
 
 func GetLatestTag(environment string) (string, error){
 	log.Println("Fetching latest tag version...")
-	arg1 := "--sort='-*authordate' | head -n1"
-	out, err := exec.Command("sh", "-c", "git tag -l --merged", environment, arg1).Output()
+	out, err := exec.Command("sh", "-c", "git describe --tags $(git rev-list --tags --max-count=1)").Output()
```

Comment:
Why did you change the git command ? what is the difference here ?

Reply:
Found a better way to get tags regardless or branches - when there were more than one tag in the old way had, i have to slice through a list and it became complicated. Using describe is the way to go

Reply:
remove environment param